src/ReverseInteger.java

```diff
@@ -0,0 +1,26 @@
+public class ReverseInteger {
+    public static int reverse(int x) {
+        long res = 0;
+        int mod = Math.abs(x);
+        int i = (int) Math.log10(mod);
+        if(i == 0 && (mod > 10 || mod < -10)) {
```

Comment:
I do not understand the purpose of this check. Can you give an example of when it is needed?

Reply:
![image](https://user-images.githubusercontent.com/44145440/177037111-e8437553-30a2-413f-acc5-46316b4b10e3.png)
Integer.MIN = -2147483648. If you apply Math.abs to this number - the minus will remain as Integer.MIN is negation of itself, thus mod can be a negative number. Although the condition can be simplified indeed to i == 0 && mod == Integer.MIN_VALUE